docs/integrations/custom-indicator.md

```diff
@@ -0,0 +1,80 @@
+---
+id: custom-indicator
+title: Custom Indicator
+description: Create a customized indicator
+---
+
+## Overview
+**CustomIndicator** is a new type of an indicator which allows you to create customized Indicators.
```

Comment:
```suggestion
**CustomIndicator** is a new type of indicator which allows you to create customized Indicators.
```

It's not really a new type, but a helper class to achieve the implementation of the custom indicator.
I think we need to modify this overview.

Reply:
fixed